test/Api.IntegrationTest/Models/AccessPoliciesResult.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Bit.Api.IntegrationTest.Models;
+
+public class AccessPoliciesResult
```

Comment:
Anything preventing us from using the API response models?

Reply:
Resolved in https://github.com/bitwarden/server/pull/2507/commits/d7d5318060d8b1c6bbe7cafb9f0d5a6cc93230aa